.github/workflows/docker.yml

```diff
@@ -38,5 +38,6 @@ jobs:
           file: scripts/Dockerfile-sync-check
           tags: |
             ghcr.io/chainsafe/sync-check:latest
+          labels: ${{ steps.meta.outputs.labels }}
```

Comment:
@samuelarogbonlo To use outputs from the `meta` step, you need a `meta` step.

Reply:
Just added the step and also change the tags for the build sync step